src/components/SearchReps.vue

```diff
@@ -150,6 +150,12 @@
 import RepresentativeCard from '@/components/RepresentativeCard.vue'
 import TakeAction from '@/components/TakeAction.vue'
 import axios from 'axios'
+//import { setupCache, buildWebStorage } from 'axios-cache-interceptor';
+
+// setting up axios with axios-cache-interceptor
+// set persistent storage to localStorage
+//const myStorage = buildWebStorage(localStorage, 'axios-cache:');
+//const axios = setupCache(Axios, {storage: myStorage, maxAge: 15 * 60 * 1000});
```

Comment:
Looks like this is leftover from something, don't forget to clean this up or finish implementation if needed